docs/reference/tools-reference/llm-tool.md

```diff
@@ -158,3 +158,69 @@ In LLM tool, the prompt is transformed to match the [openai messages](https://pl
     }
 ]
 ```
+
+### Sample 3
+This sample illustrates how to write a tool chat prompt.
+```jinja
+# system:
+You are a helpful assistant.
+
+# user:
+What is the current weather like in Boston?
+
+# assistant:
+{# The assistant message with 'tool_calls' must be followed by a tool message. #}
+## tool_calls:
+{{outputs.llm_output.tool_calls}}
+ 
+{# The tool message must follow an assistant message with 'tool_calls'. #}
+{% for item in outputs.llm_output.tool_calls %}
+# tool:
+{# 'tool_call_id's should match assistant tool_calls. #}
+## tool_call_id:
+{{item.id}}
+## content:
+{{outputs.answer}}
```

Comment:
looks weird, each tool has the same content?

Reply:
updated, please help to review